rita/src/rita_exit/db_client/mod.rs

```diff
@@ -208,12 +228,48 @@ fn client_to_db_client(
             .clone()
             .unwrap_or("".to_string()),
         country,
+        email_code: format!("{:06}", rand_code),
+        verified: false,
     }
 }
 
-fn email_ver_done(_mesh_ip: &IpAddr, _conn: &SqliteConnection) -> Result<bool, Error> {
-    //TODO actually check db
-    Ok(true)
+fn email_ver_done(client: &models::Client) -> Result<bool, Error> {
+    Ok(client.verified || SETTING.get_mailer().is_none())
```

Comment:
so if we disable mailer instant verification, I guess that's a good way to handle it.